README.md

```diff
@@ -42,6 +42,44 @@
 This library allows Java and Android applications to easily access Jellyfin servers.
 The dependencies are modular and can easily be swapped out with alternate implementations when desired.
 
+## Setup
+
+The apiclient is available through [JCenter](https://bintray.com/jellyfin/jellyfin-apiclient-java/jellyfin-apiclient-java), and thus can be installed via Gradle like any other dependency:
+
+```kotlin
+// build.gradle.kts
+repositories {
+	jcenter()
+}
+
+dependencies {
+	val apiclientVersion = "0.7.2"
```

Comment:
```suggestion
```
I think we shouldn't actually mention the version in this snippet so users won't end up using an older version if we don't update the README. The actual version to use is shown in the releases page and the badge on top of the readme.

Reply:
Right, updating the README is always a bit of a problem. Do we still keep the version variable, just with `…` in the string? I think that'd be the most obvious.

Reply:
yes